hystrix/src/main/java/com/redhat/lightblue/client/hystrix/LightblueHystrixClient.java

```diff
@@ -252,4 +254,10 @@ public ResultStream prepareFind(DataFindRequest req) throws LightblueException {
         return new ResultStream(new StreamingClosure(req),null);
     }
 
+    @Override
+    public LightblueHealthResponse lightblueHealth(LightblueHealthRequest lightblueHealthRequest)
+            throws LightblueException {
+        return null;
+    }
+
 }
```

Comment:
For consistency purpose, it would be nice to have a proper hystrix implementation of this method.